src/components/burger/style.css

```diff
@@ -0,0 +1,29 @@
+:host([visible]){
+display: block;
+height: 4em;
+transform: rotate(90deg);
+	transition: transform 100ms ease-in-out;
+}
+:host([visible]) .burger{
+	display: block;
+	width: 3.2em;	
+	height:4em;
+	padding: 1em;
+	cursor: pointer;
+}
+:host:not([visible]){
+	display: none;
+}
+.slide{
+	position: relative;
+	display: flex;
+	height: 0.4em;
+	width: auto;
+	background-color: rgb(var(--smoothly-dark-tint));
+	border-radius: 0.1em;
+	margin: 0.3em;
+}
+:host([open]){
+	transform: rotate(180deg);
```

Comment:
not entirely sure what it is we are rotating upside down but is it necessary?

Reply:
Fixed as the burger is no longer upside down